sql/size_utils.sql

```diff
@@ -78,6 +67,66 @@ BEGIN
 END;
 $BODY$;
 
+CREATE OR REPLACE FUNCTION _timescaledb_internal.range_value_to_pretty(
+    time_value      BIGINT,
+    column_type     REGTYPE
+)
+    RETURNS text LANGUAGE PLPGSQL STABLE AS
+$BODY$
+DECLARE
+BEGIN
+    IF time_value IS NULL THEN
+        RETURN format('%L', NULL);
+    END IF;
+    CASE column_type
+      WHEN 'BIGINT'::regtype, 'INTEGER'::regtype, 'SMALLINT'::regtype THEN
+        RETURN format('%L', time_value); -- scale determined by user.
+      WHEN 'TIMESTAMP'::regtype, 'TIMESTAMPTZ'::regtype THEN
+        -- assume time_value is in microsec
+        RETURN format('%1$L', _timescaledb_internal.to_timestamp(time_value)); -- microseconds
+      WHEN 'TEXT'::regtype THEN
```

Comment:
Do we allow TEXT?

Reply:
Space hash partition ranges are expressed as text.

Reply:
Actually, I might be wrong about that. I will check it out.